tests_v2/test_bell_tab_indicator.py

```
@@ -0,0 +1,253 @@
+    (workspace selection, split creation, focus changes) so the test
+    advances as soon as cmux's state has settled rather than after a
+    pessimistic delay. Avoids the flakiness CI runners would see on
+    slower hardware where 0.2-0.5s waits are not always enough.
+    """
+    deadline = time.time() + timeout
+    while time.time() < deadline:
+        try:
+            if predicate():
+                return True
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)